README.md

```diff
@@ -5,7 +5,7 @@
 [![Gem Version](https://badge.fury.io/rb/surrealist.svg)](https://rubygems.org/gems/surrealist)
 
 ![Surrealist](surrealist-icon.png)
- 
+
```

Comment:
Could you please rollback changes concerning whitespaces here? They are necessary for proper code displaying.